src/app/Instructor/apply/_components/InstructorIntroduction.tsx

```diff
@@ -0,0 +1,224 @@
+        dataName="instructorLocation"
+        title="지역"
+        required={true}
+        errors={errors.instructorLocation}
+      >
+        <Controller
+          name="instructorLocation"
+          control={control}
+          rules={{
+            required: '지역',
+          }}
+          render={({ field }) => <SelectLocation onChange={field.onChange} />}
+        />
+      </IntroductionSection>
+
+      <IntroductionSection
+        dataName="instructorGenre"
+        title="장르"
+        required={true}
+        errors={errors.instructorGenre}
+      >
+        <Controller
+          name="instructorGenre"
+          control={control}
+          defaultValue={[]}
+          rules={{
+            required: '장르',
+          }}
+          render={({ field }) => (
+            <GenreCheckboxGroup
+              onChange={field.onChange}
+              defaultValue={applyData.instructorGenre}
+            />
+          )}
+        />
+      </IntroductionSection>
+
+      <IntroductionSection
+        labelId="affiliation"
+        dataName="instructorAffiliation"
+        title="소속"
+        required={false}
+      >
+        <input
+          id="affiliation"
+          type="text"
+          className="rounded-[0.31rem] px-4 py-2 outline outline-1 outline-sub-color2 focus:outline-sub-color1"
+          placeholder="현재 속하고 있는 크루, 학원명 등을 적어주세요."
+          {...register('instructorAffiliation')}
+        />
+      </IntroductionSection>
+
+      <IntroductionSection title="SNS" required={false}>
+        <ul className="flex flex-col gap-3">
+          {SNS_ITEMS.map((item) => (
+            <SNSItem key={item.title} {...item} />
+          ))}
+        </ul>
+      </IntroductionSection>
+
+      <section className="flex w-full flex-col gap-3">
+        <label htmlFor="link0" className="mb-1">
+          <h2 className="font-bold">
+            프로필에서 보여질 강사 인스타그램 게시물의 링크를 설정해주세요.
+          </h2>
+          <p className="text-sub-color2">*최대 3개까지 표시 가능합니다.</p>
+        </label>
+
+        {Array.from({ length: 3 }).map((_, index) => (
+          <input
+            key={index}
+            id={'link' + index}
+            type="text"
+            className="h-6 flex-grow rounded-[0.31rem] px-2 py-1 outline outline-1 outline-sub-color2 focus:outline-sub-color1"
+            placeholder="게시물 주소 입력"
+            {...register('instructorInstagramLink' + index)}
+          />
+        ))}
+      </section>
+
+      <CustomEditor
+        title="강사소개"
+        dataName="applyIntroduction"
+        placeholder="어떤 목표를 두고 수업을 진행하는지, 개인적인 수업 방식, 본인의 특징, 이야기, 포부 등
+        댄스 강사로서 소개를 자유롭게 작성해주세요."
+        height="471px"
+        maxLength={1000}
+        minLength={150}
+      />
+
+      <CustomEditor
+        title="강사경력"
+        dataName="applyCareer"
+        placeholder="본인의 경력을 작성해주세요."
+        height="303px"
+        maxLength={500}
+        minLength={0}
+      />
+    </main>
+  );
+};
+
+interface IntroductionSection {
```

Comment:
```suggestion
interface IntroductionSectionProps {
```
이전 코드는 추후에 바꾸고 지금부터라도 통일해서 씁시당

Reply:
나중에 API연결할 때 바꿔주세용